etc/bin/run.sh

```diff
@@ -30,7 +30,10 @@ KERNEL_ASSEMBLY=`(cd ${PROG_HOME}/lib; ls -1 toree-kernel-assembly-*.jar;)`
 
 # disable randomized hash for string in Python 3.3+
 export PYTHONHASHSEED=0
+# The SPARK_OPTS values during installation are stored in TOREE_SPARK_OPTS. This allows values to be specified during
+# install, but also during runtime.
+export SPARK_OPTS="${__TOREE_SPARK_OPTS__} ${SPARK_OPTS}"
 
 exec "$SPARK_HOME"/bin/spark-submit \
   ${SPARK_OPTS} \
```

Comment:
Run some tests with setting --driver-java-options with multiple Java options. I was having problems setting that on SPARK_OPTS and the run.sh failing. I think we need to wrap it with quoted or some other bash-fu to get it to work. 

In my tests I was setting log4j levels and passing debug options.